src/combinator/slice.js

```diff
@@ -159,3 +159,35 @@ SkipWhileSink.prototype.event = function (t, x) {
 
   this.sink.event(t, x)
 }
+
+export function skipAfter (p, stream) {
+  return new stream.constructor(new SkipAfter(p, stream.source))
```

Comment:
Let's use `new Stream` here *for now*.  Otherwise, `skipAfter` will behave differently than other functions when `stream.constructor !== Stream`.  If we decided to switch to `stream.constructor`, then we can switch the whole library in a single PR.